### PR TITLE
Update states for Android wrapper

### PR DESCRIPTION
**Pull request by joextodd**
**Who:** Me
**Why:** Update for v3.3.4
**What:** 

- Update states for Android

### CODE REVIEW

File: android/RCTChirpConnectModule.java [hunk 1]
```diff
@@ -54,11 +54,12 @@ public RCTChirpConnectModule(ReactApplicationContext reactContext) {
...
-        constants.put("CHIRP_CONNECT_STATE_PAUSED", 1);
-        constants.put("CHIRP_CONNECT_STATE_RUNNING", 2);
-        constants.put("CHIRP_CONNECT_STATE_SENDING", 3);
-        constants.put("CHIRP_CONNECT_STATE_RECEIVING", 4);
+        constants.put("CHIRP_CONNECT_NOT_CREATED", 0);
+        constants.put("CHIRP_CONNECT_STATE_STOPPED", 1);
+        constants.put("CHIRP_CONNECT_STATE_PAUSED", 2);
+        constants.put("CHIRP_CONNECT_STATE_RUNNING", 3);
+        constants.put("CHIRP_CONNECT_STATE_SENDING", 4);
+        constants.put("CHIRP_CONNECT_STATE_RECEIVING", 5);
```
Comment (dinu88) [inline android/RCTChirpConnectModule.java]: A bit confused where and how this is used? From my understanding, this states should reflect `ConnectState` however, don't see this to be used anywhere in the file?

Comment (joextodd) [reply android/RCTChirpConnectModule.java]: This is some React Native magic that makes these constants available in JavaScript land. See https://facebook.github.io/react-native/docs/native-modules-android.html

Comment (dinu88) [reply android/RCTChirpConnectModule.java]: yes but this is just some strings pointed to numbers, how this reflects `ConnectState` values?

Comment (joextodd) [reply android/RCTChirpConnectModule.java]: It's just like an enumeration really, in `App.js` you can then use these like so 

`if event.status == ChirpConnect.CHIRP_CONNECT_STATE_STOPPED`

**Comment by remintz** [comment]
This change has caused an execution exception. Please see attached. Rolled back to using literal integers.
img_3173

**Comment by joextodd** [comment]
@remintz could you confirm that you are using v3.3.4 aar file?

**Comment by joextodd** [comment]
@remintz thanks for raising this issue, I must have been testing with a cached version. Submitted a fix for this at #4

**Pull request merged**